### PR TITLE
Update Wordpress libraries, webpack, and related dependencies

### PR DESCRIPTION
**Pull request by kmonahan**
Updates the Wordpress libraries to their latest versions, Webpack to Webpack 5, and related dependencies as needed to get everything running locally.

### CODE REVIEW

File: src/cli/webpack.config.js [hunk 1]
```diff
@@ -28,17 +28,10 @@ const config = {
...
 		historyApiFallback: true,
 	},
 	resolve: {
-		modules: [
-			path.relative(
-				process.cwd(),
-				path.resolve( __dirname, '../../node_modules' )
-			),
-			path.resolve( process.cwd(), 'node_modules' ),
-		],
```
Comment (youknowriad) [inline src/cli/webpack.config.js:41]: Thanks for the PR, this is the only thing that I'm not sure about, the idea was that I wanted to ensure the modules from blockbook (WP modules) have more priority than the ones that come from the repo of the theme/plugin. Do you know if this is still the case and how is this achieved?

Comment (kmonahan) [reply src/cli/webpack.config.js:41]: The issue I was hitting here is that with this included, webpack was not resolving nested node_modules. For example, `react-autosize-textarea`, which was installed in `blockbook/node_modules/@wordpress/block-editor/node_modules` wasn't being found.

Comment (kmonahan) [reply src/cli/webpack.config.js:41]: My read of https://nodejs.org/docs/latest-v14.x/api/modules.html#modules_loading_from_node_modules_folders is that Node starts with the parent of the current location and works outward only if the module isn't found, so I think blockbook modules would have priority over ones in the theme or plugin.

**Comment by youknowriad** [comment]
Ok I'm merging this let's give this a try.

**Pull request merged**